styles/feedreader.css

```diff
@@ -0,0 +1,17 @@
+.get_involved {
+    padding-top: 60px; margin-top: -60px;
+}
+
+.get_involved__title{
+    color:#B6B6B6;
+    font-weight: normal;
+    line-height: 1;
+    font-size: 32px;
+    font-family: "Raleway", "Helvetica Neue", Helvetica, Arial, sans-serif;
+}
+
+.get_involved__description{
+    font-size:125%;
+    margin-top: 12px;
+    text-align: justify;
+}
```

Comment:
It's strange that only this one section has custom overrides like this. Why does this section need to have that specific font and justified test but the others don't? The more we can just use the same style for everything, the easier this will be to maintain.

Reply:
I don't recall if BS 3 still has the option of getting just the minimmum (i.e., without JS and get just some components, like grid and buttons). If so, you can still use a minumal BS code.